diff -ur accel-ppp-1.6.1/accel-pppd/auth/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/auth/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/auth/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/auth/CMakeLists.txt	2012-04-17 15:08:00.634062871 +0200
@@ -8,6 +8,6 @@
 TARGET_LINK_LIBRARIES(auth_mschap_v2 ${crypto_lib})
 
 INSTALL(TARGETS auth_pap auth_chap_md5 auth_mschap_v1 auth_mschap_v2
-	LIBRARY DESTINATION lib/accel-ppp
+	LIBRARY DESTINATION lib64/accel-ppp
 )
 
Tylko w accel-ppp-1.6.1.new/accel-pppd/auth: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/ctrl/l2tp/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/ctrl/l2tp/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/ctrl/l2tp/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/ctrl/l2tp/CMakeLists.txt	2012-04-17 15:08:47.786295511 +0200
@@ -10,7 +10,7 @@
 )
 #TARGET_LINK_LIBRARIES(l2tp nl nl-genl)
 
-INSTALL(TARGETS l2tp LIBRARY DESTINATION lib/accel-ppp)
+INSTALL(TARGETS l2tp LIBRARY DESTINATION lib64/accel-ppp)
 
 FILE(GLOB dict "${CMAKE_CURRENT_SOURCE_DIR}/dict/*")
 INSTALL(FILES ${dict} DESTINATION share/accel-ppp/l2tp)
Tylko w accel-ppp-1.6.1.new/accel-pppd/ctrl/l2tp: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/ctrl/pppoe/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/ctrl/pppoe/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/ctrl/pppoe/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/ctrl/pppoe/CMakeLists.txt	2012-04-17 15:09:26.651469094 +0200
@@ -18,4 +18,4 @@
 #set_property(TARGET pppoe PROPERTY INSTALL_RPATH_USE_LINK_PATH FALSE)
 #set_property(TARGET pppoe PROPERTY INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib/accel-ppp)
 
-INSTALL(TARGETS pppoe LIBRARY DESTINATION lib/accel-ppp)
+INSTALL(TARGETS pppoe LIBRARY DESTINATION lib64/accel-ppp)
Tylko w accel-ppp-1.6.1.new/accel-pppd/ctrl/pppoe: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/ctrl/pptp/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/ctrl/pptp/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/ctrl/pptp/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/ctrl/pptp/CMakeLists.txt	2012-04-17 15:09:40.922144806 +0200
@@ -2,4 +2,4 @@
 
 ADD_LIBRARY(pptp SHARED pptp.c)
 
-INSTALL(TARGETS pptp LIBRARY DESTINATION lib/accel-ppp )
+INSTALL(TARGETS pptp LIBRARY DESTINATION lib64/accel-ppp )
Tylko w accel-ppp-1.6.1.new/accel-pppd/ctrl/pptp: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/extra/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/extra/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/extra/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/extra/CMakeLists.txt	2012-04-17 15:10:01.156436224 +0200
@@ -8,7 +8,7 @@
 ADD_LIBRARY(connlimit SHARED connlimit.c)
 
 INSTALL(TARGETS pppd_compat ippool ipv6pool sigchld chap-secrets logwtmp connlimit
-	LIBRARY DESTINATION lib/accel-ppp
+	LIBRARY DESTINATION lib64/accel-ppp
 )
 
 IF (SHAPER)
@@ -18,7 +18,7 @@
 		SET_PROPERTY(TARGET shaper_tbf PROPERTY LINK_FLAGS "-L${LIBNL_PREFIX}/lib")
 	ENDIF (LIBNL_PREFIX)
 	TARGET_LINK_LIBRARIES(shaper_tbf nl m)
-	INSTALL(TARGETS shaper_tbf LIBRARY DESTINATION lib/accel-ppp)
+	INSTALL(TARGETS shaper_tbf LIBRARY DESTINATION lib64/accel-ppp)
 ENDIF (SHAPER)
 
 
Tylko w accel-ppp-1.6.1.new/accel-pppd/extra: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/extra/net-snmp/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/extra/net-snmp/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/extra/net-snmp/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/extra/net-snmp/CMakeLists.txt	2012-04-17 15:10:16.977185330 +0200
@@ -25,5 +25,5 @@
 ENDIF (NETSNMP_PREFIX)
 
 INSTALL(TARGETS net-snmp
-	LIBRARY DESTINATION lib/accel-ppp
+	LIBRARY DESTINATION lib64/accel-ppp
 )
Tylko w accel-ppp-1.6.1.new/accel-pppd/extra/net-snmp: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/ipv6/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/ipv6/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/ipv6/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/ipv6/CMakeLists.txt	2012-04-17 15:10:28.777744083 +0200
@@ -2,6 +2,6 @@
 ADD_LIBRARY(ipv6_nd SHARED nd.c)
 
 INSTALL(TARGETS ipv6_dhcp ipv6_nd
-	LIBRARY DESTINATION lib/accel-ppp
+	LIBRARY DESTINATION lib64/accel-ppp
 )
 
Tylko w accel-ppp-1.6.1.new/accel-pppd/ipv6: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/logs/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/logs/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/logs/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/logs/CMakeLists.txt	2012-04-17 15:10:57.472436098 +0200
@@ -15,21 +15,21 @@
 	ADD_LIBRARY(log_file SHARED log_file.c)
 	TARGET_LINK_LIBRARIES(log_file rt)
 	INSTALL(TARGETS log_file
-		LIBRARY DESTINATION lib/accel-ppp
+		LIBRARY DESTINATION lib64/accel-ppp
 	)
 ENDIF(LOG_FILE)
 
 IF(LOG_TCP)
 	ADD_LIBRARY(log_tcp SHARED log_tcp.c)
 	INSTALL(TARGETS log_tcp
-		LIBRARY DESTINATION lib/accel-ppp
+		LIBRARY DESTINATION lib64/accel-ppp
 	)
 ENDIF(LOG_TCP)
 
 IF(LOG_SYSLOG)
 	ADD_LIBRARY(log_syslog SHARED log_syslog.c)
 	INSTALL(TARGETS log_syslog
-		LIBRARY DESTINATION lib/accel-ppp
+		LIBRARY DESTINATION lib64/accel-ppp
 	)
 ENDIF(LOG_SYSLOG)
 
@@ -37,7 +37,7 @@
 	ADD_LIBRARY(log_pgsql SHARED log_pgsql.c)
 	TARGET_LINK_LIBRARIES(log_pgsql pq)
 	INSTALL(TARGETS log_pgsql
-		LIBRARY DESTINATION lib/accel-ppp
+		LIBRARY DESTINATION lib64/accel-ppp
 	)
 ENDIF(LOG_PGSQL)
 
Tylko w accel-ppp-1.6.1.new/accel-pppd/logs: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/radius/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/radius/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/radius/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/radius/CMakeLists.txt	2012-04-17 15:11:14.459907114 +0200
@@ -15,7 +15,7 @@
 ADD_LIBRARY(radius SHARED ${sources})
 
 INSTALL(TARGETS radius
-	LIBRARY DESTINATION lib/accel-ppp
+	LIBRARY DESTINATION lib64/accel-ppp
 )
 FILE(GLOB dict "${CMAKE_CURRENT_SOURCE_DIR}/dict/*")
 INSTALL(FILES ${dict} DESTINATION share/accel-ppp/radius)
Tylko w accel-ppp-1.6.1.new/accel-pppd/radius: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/shaper/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/shaper/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/shaper/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/shaper/CMakeLists.txt	2012-04-17 15:11:24.817064190 +0200
@@ -1,6 +1,6 @@
 ADD_LIBRARY(shaper SHARED shaper.c limiter.c tc_core.c libnetlink.c)
 
 INSTALL(TARGETS shaper
-	LIBRARY DESTINATION lib/accel-ppp
+	LIBRARY DESTINATION lib64/accel-ppp
 )
 
Tylko w accel-ppp-1.6.1.new/accel-pppd/shaper: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/accel-pppd/triton/CMakeLists.txt accel-ppp-1.6.1.new/accel-pppd/triton/CMakeLists.txt
--- accel-ppp-1.6.1/accel-pppd/triton/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/accel-pppd/triton/CMakeLists.txt	2012-04-17 15:11:42.714578295 +0200
@@ -30,11 +30,11 @@
 
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
 
-ADD_DEFINITIONS(-DMODULE_PATH="${CMAKE_INSTALL_PREFIX}/lib/accel-ppp")
+ADD_DEFINITIONS(-DMODULE_PATH="${CMAKE_INSTALL_PREFIX}/lib64/accel-ppp")
 
 ADD_LIBRARY(triton SHARED ${sources_c})
 TARGET_LINK_LIBRARIES(triton dl)
 
 INSTALL(TARGETS triton
-	LIBRARY DESTINATION lib/accel-ppp
+	LIBRARY DESTINATION lib64/accel-ppp
 )
Tylko w accel-ppp-1.6.1.new/accel-pppd/triton: CMakeLists.txt~
diff -ur accel-ppp-1.6.1/crypto/CMakeLists.txt accel-ppp-1.6.1.new/crypto/CMakeLists.txt
--- accel-ppp-1.6.1/crypto/CMakeLists.txt	2012-03-20 06:43:40.000000000 +0100
+++ accel-ppp-1.6.1.new/crypto/CMakeLists.txt	2012-04-17 15:11:52.955063178 +0200
@@ -11,7 +11,7 @@
 	ADD_LIBRARY(internal-crypto SHARED ${sources})
 
 	INSTALL(TARGETS internal-crypto
-		LIBRARY DESTINATION lib/accel-ppp
+		LIBRARY DESTINATION lib64/accel-ppp
 	)
 
 ENDIF (CRYPTO STREQUAL INTERNAL)
Tylko w accel-ppp-1.6.1.new/crypto: CMakeLists.txt~
