--- accel-ppp-1.13.0/accel-pppd/extra/net-snmp/sessionTable.h.orig	2024-03-30 18:58:28.000000000 +0100
+++ accel-ppp-1.13.0/accel-pppd/extra/net-snmp/sessionTable.h	2025-04-10 20:03:26.406324994 +0200
@@ -22,11 +22,11 @@ extern "C" {
 
 /* other required module components */
     /* *INDENT-OFF*  */
-config_add_mib(ACCEL-PPP-MIB)
-config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_interface)
-config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_data_access)
-config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_data_get)
-config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_data_set)
+config_add_mib(ACCEL-PPP-MIB);
+config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_interface);
+config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_data_access);
+config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_data_get);
+config_require(ACCEL-PPP-MIB/sessionTable/sessionTable_data_set);
     /* *INDENT-ON*  */
 
 /* OID and column number definitions for sessionTable */
